test/src/e2e_vm_tests/test_programs/should_pass/language/enum_padding/test.toml

```diff
@@ -0,0 +1,3 @@
+category = "run"
+expected_result = { action = "return_data", value = [ 0xce, 0x55, 0xff, 0x05, 0x11, 0x3a, 0x24, 0x2e, 0xc7, 0x9a, 0x23, 0x75, 0x0c, 0x7e, 0x2b, 0xab, 0xaf, 0x98, 0xa8, 0xdc, 0x41, 0x66, 0x90, 0xc8, 0x57, 0xdd, 0x31, 0x72, 0x0c, 0x74, 0x82, 0xb6 ] }
```

Comment:
The value here is in different encoding than in the README.md. Should the README.md be updated to reflect this?